node/device/tun/tun_windows.go

```diff
@@ -85,9 +85,12 @@ func nanotime() int64
 // interface with the same name exist, it is reused.
 func NewTUN() (Device, error) {
 	// Remove the previous adapter
-	if err := wintun.Uninstall(); err != nil {
-		return nil, err
-	}
+	// TODO: Uninstall() fails in some machines. So there are indeed errors returned
+	// sometimes in these cases. However in order not to block following steps,
+	// the errors are ignored for now. Deep dive needed.
+	//
+	// tracking issue: https://github.com/pairmesh/pairmesh/issues/65
+	wintun.Uninstall()
```

Comment:
```suggestion
	_ = wintun.Uninstall()
```
Prefer to ignore it explicitly to make others aware it.